src/libstd/lib.rs

```diff
@@ -80,8 +80,8 @@
 //!
 //! Note the documentation for the primitives [`str`] and [`[T]`][slice] (also
 //! called 'slice'). Many method calls on [`String`] and [`Vec<T>`] are actually
-//! calls to methods on [`str`] and [`[T]`][slice] respectively, via [deref
-//! coercions].
+//! calls to methods on [`str`] and [`[T]`][slice] respectively, via
+//! [deref-coercions].
```

Comment:
I added it because the tidy check does not deal with spaces in named links, not realising that it was the short form. Fixed.